nml/expression/functioncall.py

```diff
@@ -552,6 +552,15 @@ def builtin_trigonometric(name, args, pos):
     return ConstantFloat(trigonometric_func_table[name](val.value), val.pos)
 
 
+@builtin
+def builtin_sqrt(name, args, pos):
+    if len(args) == 0:
+        raise generic.ScriptError("sqrt() requires 1 argument", pos)
+    elif len(args) != 1:
+        raise generic.ScriptError("sqrt() requires only 1 argument", pos)
+    return ConstantFloat(math.sqrt(args[0].value),pos)
```

Comment:
This probably needs a check for args[0] being a constant value, similar as for the trigonometric functions above.
```
if not isinstance(val, (ConstantNumeric, ConstantFloat)):
        raise generic.ScriptError("Parameter for " + name + "() must be a constant", pos)
```

Actually, it may be easier to just extend the mapping in builtin_trigonometric, and rename it to builtin_math or something.

Reply:
Updated Pull Request's commit with proposed updates